crates/topos-sequencer-subnet-runtime/src/proxy.rs

```diff
@@ -122,11 +122,17 @@ impl SubnetRuntimeProxy {
                                     "Source head certificate id received {:?}",
                                     certificate_and_position
                                 );
-                                // If the position is not provided, it should start form -1, so that first fetched is subnet genesis block
+                                // If tce source head position is provided, continue syncing from it
```

Comment:
```suggestion
                                // If tce source head position is provided, continue synching from it
```

Reply:
I feel that `synching` isn't really standard in computer industry. I would prefer sticking to `syncing` or even `synchronizing`

Reply:
Both spellings are correct-ish. Let's stick to "synchronizing" then.